src/dof.cpp

```diff
@@ -96,7 +96,7 @@ void DOF::updateMinMax()
   DBGP("Joint 0 min "  << minq << " max " << maxq);
   if (maxq < minq) std::swap(maxq, minq);  
   DBGP("maxq " << maxq << " minq " << minq);
-  std::list<Joint *>::iterator j;
+  std::vector<Joint *>::iterator j;
```

Comment:
I kept getting some weird runtime errors while using the list iterator that I couldn't figure out for the life of me.  I can try to switch it back to a list.